frontend/src/Resources/actionCreator.js

```diff
@@ -0,0 +1,22 @@
+import { SERVER_URL } from 'react-native-dotenv';
+
+export const FETCH_RESOURCES_REQUEST = 'fetchResourcesRequest';
+export const FETCH_RESOURCES_SUCCESS = 'fetchResourcesSuccess';
+export const FETCH_RESOURCES_FAILURE = 'fetchResourcesFailure';
+
+const URL = `http://${SERVER_URL}/kingdom/resources`;
+
+export function fetchResources() {
+  return (dispatch) => {
+    dispatch({ type: FETCH_RESOURCES_REQUEST });
+    fetch(URL)
+      .then((response) => {
+        if (response.status === 200) {
+          return response.json();
+        }
+        throw new Error(response.status);
```

Comment:
you want to show this to the user? or what are you doing with this status code, where do you use it?

Reply:
I think it will direct to the error page which hasn't been implemented yet